docs/recipes/service.md

```diff
@@ -53,6 +53,9 @@ Bake.New
 |                    | With Method                      |                                    |
 | Communication      | :white_check_mark: HTTP          | :white_check_mark: Mock            |
 | Core               | :white_check_mark: Dotnet        | :white_check_mark: Mock            |
+| Cors(s)            | :white_check_mark:               | :no_entry:                         |
+|                    | Disabled                         |                                    |
+|                    | AspNetCore                       |                                    |
```

Comment:
in this table we list only the default, not all the options. default is set to `Disabled` in service recipe, remove `AspNetCore` from here